### ISSUE TITLE
compiling efibootmgr now requires -ldl

### ISSUE THREAD

**Issue by cbrune** [rhinstaller/efibootmgr#30]
With the latest efivar and efibootmgr, efibootmgr fails to compile due to undefined references to dlopen() and friends.

It looks like -ldl was added to efivar recently which causes this:
```
/work/monster-04/curt/trees/onie-cbrune/build/kvm_x86_64-r0/dev-sysroot/usr/lib/../lib64/libefivar.so: undefined reference to `dlclose'  
/work/monster-04/curt/trees/onie-cbrune/build/kvm_x86_64-r0/dev-sysroot/usr/lib/../lib64/libefivar.so: undefined reference to `dlopen'  
/work/monster-04/curt/trees/onie-cbrune/build/kvm_x86_64-r0/dev-sysroot/usr/lib/../lib64/libefivar.so: undefined reference to `dlsym'
collect2: error: ld returned 1 exit status
make[1]: *** [src/efibootmgr/efibootmgr] Error 1
```

Adding `-ldl` to `LIBS` in src/efibootmgr/module.mk fixes the problem.

**Comment by vathpela** [comment]
This is fixed in the "dp" branch with commit ada1a1cf .  The "dp" branch will be the next release.

**Comment by vathpela** [comment]
(And thanks for spotting this problem!)
